lib/query/standard-query.js

```diff
@@ -0,0 +1,33 @@
+const _ = require('lodash')
+const { filterAndTransform } = require('../filter-and-transform')
+const { params: createSqlParams } = require('../sql-query-builder')
+const packageFeatures = require('./package-features')
+
+function standardQuery (features, sqlString, options = {}) {
+  const { limit } = options
+  const params = createSqlParams(features, options)
+  const filtered = filterAndTransform(sqlString, params)
+
+  if (options.skipLimitHandling || !options.limit) {
+    return packageFeatures(filtered, options)
+  }
+
+  // options.limit is incremented by one in normalizeOptions.js; if filtered.length === options.limit, original limit option has been exceeded
+  const limitExceeded = filtered.length === limit
```

Comment:
I don't understand the logic here. Why is `filtered.length === limit`, not `filtered.length >= limit`?

Reply:
I think that would work fine as well.  But if there was a `limit` option, winnow should have only returned an array of features equal to the `limit` + 1 value.